packages/react-mui-store-file-viewer/src/components/StoreDataGrid/FolderPath.tsx

```diff
@@ -46,9 +48,13 @@ const FolderPath: FC<Props> = ({ useRootIcon, defaultHidden }) => {
         separator={<NavigateNextIcon fontSize="small" />}
         aria-label="breadcrumb"
       >
-        {useRootIcon ? (
+        {true ? (
```

Comment:
This basically makes the ternary useless? Could just drop the `<FolderOption>` further down?

Reply:
Ya it does, I left thinking it maybe the design decision would get changed back, but it is so little code I'll clean it out